### DISCUSSION TITLE
[Feature] Add DNS history source category

### DISCUSSION THREAD

**Discussion by M09Ic** [projectdiscovery/subfinder#494] [Ideas]
I want to find the dns history reslove for domain.

We can add a history category, which is used to manage dns history sources. And add SecurityTrails/Spyse/DNS History and other data sources to get the history data,  mark the results as ishistory flag.

**Comment by ehsandeep** [comment]
@M09Ic moved the discussion to ideas, as this is something we may explore in the future.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
